releaseNotes/AzureDevOpsPushPipelineExtension.md

```diff
@@ -2,6 +2,13 @@
 Azure App Configuration Push extension for Azure DevOps pipeline can be installed from the [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task-push). The version information can be found in the console when the task is executed:
 ![sample](pictures/AzureDevOpsPushExtensionVersionSample.PNG)
 
+### v1.4.4 - July, 22 2021
+* Added the capability to use certificate based authentication.
+* Added logs that show more information about the task.
+* Marked extension as public.

```

Comment:
This can be confusing as the extension has always been publicly accessible. Maybe something like this? And move it to the first?
* The Azure App Configuration Push pipeline task is now generally available.